collection.php

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang='en'>
+<head>
+    <meta charset='UTF-8'/>
+    <meta name='viewport' content='width=device-width, initial-scale=1.0'/>
+    <title>Title</title>
+    <link rel='stylesheet' href='normalize.css'/>
+</head>
+<body>
+    <form method='POST'>
+        <input name='name' type='text'/>
```

Comment:
Can you label the inputs so you know the user is filling in data in the right place?